docs/installing-oggm.rst

```diff
@@ -127,7 +127,11 @@ Packages
 
 Install the packages from the `conda-forge`_ channel::
 
-    conda install -c conda-forge geopandas matplotlib Pillow joblib netCDF4 scikit-image configobj nose pyproj numpy krb5 rasterio xarray filelock
+    conda install -c oggm -c conda-forge oggm-deps
+
+If you want to install oggm itself to just use it, install oggm instead of oggm-deps.
+oggm-deps is a meta package that contains no actual content, and just depends on all
+dependencies of oggm.
 
```

Comment:
There are two packages for oggm on conda.
oggm and oggm-deps.
oggm contains the actual oggm code, as you get if you run pip install on oggm.
oggm-deps does not contain any code, and only pulls in the required dependencies to run oggm, so you can use your own fork of it.